tensorboard/components/tf_card_heading/tf-card-heading.html

```diff
@@ -168,6 +171,11 @@
         this.$.descriptionDialog.positionTarget = e.target;
         this.$.descriptionDialog.toggle();
       },
+
+      // Break the string at natural points, including commas, equals, and slashes
```

Comment:
Why equals? They might be used to encode hyper parameters. If I have a run name like `learning_rate=0.1,filters=64,batch_size=100` then it would be nice if made an attempt to show me one association per line.

Reply:
I just copied the same set of characters used for run breaking in tf-runs-selector.html, since it seemed like it was worth having the same behavior.  If you like I can restrict one or both to not break on equals signs though.